CMakeLists.txt

```
@@ -41,7 +41,7 @@ endif(MSVC)
 
 configure_file( version_info.in version_info.h @ONLY)
 
-add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/lua EXCLUDE_FROM_ALL)
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/lua)
```

```suggestion
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/lua)
```
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/lua EXCLUDE_FROM_ALL)